cli/src/pcluster/config/cluster_config.py

```diff
@@ -23,6 +23,11 @@
 from pcluster.aws.common import get_region
```

Comment:
I'll leave this general comment here: shall we have some unit test for byos config validation, especially to verify that bad values are properly handled?